### PR TITLE
adds json tags for ZDBResult struct

### PR DESCRIPTION
**Pull request by mariobassem**
## Related issues:

- #1952

### CODE REVIEW

**Review by xmonader — CHANGES_REQUESTED** [review]
Looks good, however this is a breaking change for all of the clients in typescript and and go, won't merge until there're ready PRs in both of the sdks

**Review by muhamadazmy — APPROVED** [review]

**Comment by muhamadazmy** [comment]
Please make sure that @xmonader comment is handled before merging. I checked in Golang side and this does not break the Go code (decoding data with default tags) but need to be handled in the right way in the other clients

**Pull request closed without merging**
